app/renderer/js/preload.js

```diff
@@ -55,6 +56,23 @@ document.addEventListener('DOMContentLoaded', () => {
 			ipcRenderer.send('forward-message', 'reload-viewer');
 		});
 	}
+
+	const { $, lightbox } = window;
```

Comment:
Not sure but should we use the JS here instead of the jQuery?

Reply:
@abhigyank can you add few more lines here about what's going on here? Something like -
> Open image attachment link in the lightbox instead of opening in the default browser

Reply:
Does that make any major difference?

Reply:
Yeah, I think jQuery would make this very easy whereas js we would need query all `.message_content p a` and add listener. And event bubbling would also be hard since the target would be `a` and we need to then check if its inside `.message_content p` sounds too much for doing this in js side.

Otherwise, i would have said it should be on js side for consistency